tests/m2m_through_regress/tests.py

```diff
@@ -251,6 +260,43 @@ def test_remove_reverse(self):
         self.assertQuerysetEqual(
             self.driver.car_set.all(), [])
 
+    def test_remove_empty(self):
+        c = Car.objects.create(make=None)
+        c.save()
+        with self.assertRaisesMessage(
+            ValueError,
+            '"<Car: None>" needs to have a value for field "car" '
+            'before this many-to-many relationship can be used.'
+        ):
+            c.drivers._remove_items('car', 'drivers', self.driver)
+
+    def test_clear_empty(self):
+        c = Car.objects.create(make=None)
+        c.save()
```

Comment:
save is not needed : objects.create returns an already saved instance. Tests succeed without the save